platform/android/os_android.h

```diff
@@ -93,6 +93,7 @@ class OS_Android : public OS_Unix {
 
 private:
 	Vector<TouchPos> touch;
+	Point2 prev_pos;
```

Comment:
Can you use a more descriptive name for the field, and add some comments to describe its use.